### ISSUE TITLE
Test coverage

### ISSUE THREAD

**Issue by blueyed** [opbeat/opbeat_python#174]
Are there plans to setup something like Codecov to track code coverage?

**Comment by beniwohli** [comment]
Yes, amongst other things :) at the moment, we're in the process of preparing to move this repository to the Elastic organization. After that, we plan to do some cleanup of the test suite and add coverage tracking.
